### PR TITLE
📊 ai: AI index report - Policy & Governance + AI PhDs

### PR DESCRIPTION
**Pull request by veronikasamborska1994**
hi @antea04 ! a lot of this code has already been reviewed by Pablo A previously and it's generally v straightforward so I thought it might be a good one for you to review :) let me know if you don't have time for it this week though!

### CODE REVIEW

**Review by antea04 — APPROVED** [review]
Hi Veronika!

The PR looks great - I left a few nitpicky comments regarding the metadata, but you can decide if you want to implement them. If you disagree with something, feel free to let me know, then I can adjust my reviews going forward ☺️ 

One more thing thats not in the code (nitpick again): on [this chart](http://staging-site-ai-index-cont2/grapher/national-strategies-on-artificial-intelligence) the grey colour for "in development" has changed between the updates and now it looks (at least on my screen) _really_ similar to the blue. If you agree, maybe you could change it back to the lighter color.

Otherwise feel free to merge 👍  Thanks for trusting me with the PR review!

File: snapshots/artificial_intelligence/2024-06-28/ai_phds.csv.dvc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    description: |-
+      The AI Index Report tracks, collates, distills, and visualizes data related to artificial intelligence (AI). The mission is to provide unbiased, rigorously vetted, broadly sourced data to enable policymakers, researchers, executives, journalists, and the general public to develop a more thorough and nuanced understanding of the complex field of AI.
+    date_published: "2024"
+    title_snapshot: AI Index Report - AI PhDs
+
+    description_snapshot: |-
+      Data from the Computing Research Association’s Taulbee Survey via the AI Index report, which evaluates the state of CS and AI postsecondary education in the United States and Canada. The survey covers 297 PhD-granting CS departments across the United States and Canada.
+    # Citation
+    producer: CRA Taulbee Survey via AI Index
+    citation_full: |-
```
[the remark this answers is not in the document]
Comment (veronikasamborska1994) [reply snapshots/artificial_intelligence/2024-06-28/ai_phds.csv.dvc:16]: @antea04 I actually haven't been able to stick around for the entire meeting recently about licenses with the lawyers. is this something they mentioned we should do? I've always used the licence for the actual source that we get the data from but not sure if the guidelines have changed?

File: etl/steps/data/garden/artificial_intelligence/2024-06-28/ai_phds.meta.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Artificial Intelligence
+
+      grapher_config:
+              note:  Since 2004, total response rates for the Taulbee Survey have ranged between 61–83%. This means that not every graduate is included, and the share who are varies from year to year.
+    display:
+      numDecimalPlaces: 0
```
[the remark this answers is not in the document]
Comment (veronikasamborska1994) [reply etl/steps/data/garden/artificial_intelligence/2024-06-28/ai_phds.meta.yml:12]: good point! thanks for spotting this!

**Comment by veronikasamborska1994** [comment]
hi @antea04! thanks a lot for taking a look at the PR so promptly! you've nicely spotted the things that could be improved in the metadata and the wrong license :) I've implemented all your suggestions, so there's no disagreement on my part. Well done!  I've also been away for a bit, so I'm not sure what the guidelines are regarding citing licenses when we are sourcing data indirectly. I'll keep it as it is for now since it's worked well in the past. However, if you have any recent information that suggests I should change it, please let me know!

**Comment by antea04** [comment]
> hi @antea04! thanks a lot for taking a look at the PR so promptly! you've nicely spotted the things that could be improved in the metadata and the wrong license :) I've implemented all your suggestions, so there's no disagreement on my part. Well done! I've also been away for a bit, so I'm not sure what the guidelines are regarding citing licenses when we are sourcing data indirectly. I'll keep it as it is for now since it's worked well in the past. However, if you have any recent information that suggests I should change it, please let me know!

Thanks! I don't have any specific information on citing sources indirectly, so I think sticking to what worked in the past is the way to go ☺️

**Pull request merged**